### DISCUSSION TITLE
How to implement autocomplete?

### DISCUSSION THREAD

**Discussion by burningyouth** [tamagui/tamagui#743] [Q&A]
Hi there! I’m wondering how I can fix a problem I’m having. The issue is that the select popover isn't positioned below the trigger. It opens either where the value is or it covers the trigger. This creates a problem when using it with an async select or autocomplete, as the items aren’t always the same each time, or when pagination is used. Additionally, I want to be able to see the value I’m typing inside the trigger. Can you assist me with this?

**Comment by la55u** [comment]
you can take inspiration from here: https://tamagui-extras.vercel.app/?path=/story/ui-forms-autocomplete--single

**Reply by burningyouth** [reply]
Thank you! I suppose this is the only way of dealing with this issue. But I was thinking that there might be a better approach than manually handling state and aria attributes on your own. And I can’t use your components since they have a less flexible interface.
